frameworks/Scala/scruffy/src/test/scala/BenchmarkVerify.scala

```diff
@@ -11,7 +11,7 @@ object BenchmarkVerify extends App {
 
   val sw = Stopwatch()
 
-  val futures = for ( k <- 1 to 10000 ) yield {
+  val futures = for ( k <- 1 to 30000 ) yield {
```

Comment:
Just creating 30,000 deferred requests to verify the endpoints work as expected when a change is made. It's code in the src/test source folder so isn't include in the JAR that is built when the build script is invoked.